tests/test_for_suite/test_for_cli/test_for_runner.py

```diff
@@ -93,3 +134,75 @@ def test_run_skip_unchanged_true(runner, mock_repo):
     number_of_result_files = len(os.listdir(join(mock_repo[0], 'results')))
     os.chdir(previous_wd)
     assert number_of_result_files == 3
+
+
+def test_new(runner, tmpdir):
+    """Expect memote new to create a cookiecutter repo."""
+    target_dir = str(tmpdir)
+    user_responses = "John\nj@d.com\nJD\nmock-repo\nmock-repo\n" \
+                     "description\n2019-02-07\n2019\n0.1.0\n" \
+                     "default\ndefault\ngh-pages"
+    result = runner.invoke(cli, [
+        "new", "--directory", target_dir], input=user_responses)
+    assert result.exit_code == 0
+
+
+def test_online(runner, mock_repo, monkeypatch):
+    # Build-up
+    def mock_setup_gh_repo(github_repository, github_username, note):
```

Comment:
You could use a catch all since you're not interested in the arguments.

```suggestion
    def mock_setup_gh_repo(*args, **kwargs):
```

Reply:
Done